policies/templates/gcp_glb_forwarding_rules_whitelist.yaml

```diff
@@ -0,0 +1,155 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-glb-forwarding-rule-whitelist-v1
+  annotations:
+    description: "Constraint to restrict forwarding rules based on a whitelist"
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGLBForwardingRuleWhitelistConstraintV1
+        plural: gcpglbforwardingrulewhitelistconstraintsv1
+      validation:
+        openAPIV3Schema:
+          whitelist:
+            instances:
+              type: array
+              items: 
+                type: object
+                properties:
+                  target:
+                    type: string
+                  ip_address:
+                    type: string
+                  ip_protocol:
+                    type: string
+                  port_range:
+                    type: string
+                  load_balancing_scheme:
+                    type: string
+                required:
+                  - target
+  targets:
+    validation.gcp.forsetisecurity.org:
+      rego: | #INLINE("validator/gcp_glb_forwarding_rules_whitelist.rego")
+           #
+           # Copyright 2018 Google LLC
+           #
+           # Licensed under the Apache License, Version 2.0 (the "License");
+           # you may not use this file except in compliance with the License.
+           # You may obtain a copy of the License at
+           #
+           #      http://www.apache.org/licenses/LICENSE-2.0
+           #
+           # Unless required by applicable law or agreed to in writing, software
+           # distributed under the License is distributed on an "AS IS" BASIS,
+           # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+           # See the License for the specific language governing permissions and
+           # limitations under the License.
+           #
+           
+           package templates.gcp.GCPGLBForwardingRulesWhitelistConstraintV1
+           
+           import data.validator.gcp.lib as lib
+           
+           ###########################
+           # Find allowlist Violations
+           ###########################
+           deny[{
+           	"msg": message,
+           	"details": metadata,
+           }] {
+           	constraint := input.constraint
+           	lib.get_constraint_params(constraint, params)
+           	asset := input.asset
+           
+           	asset.asset_type == "compute.googleapis.com/ForwardingRule"
```

Comment:
I think `compute.googleapis.com/GlobalForwardingRule` is also being scanned in the Forseti forwarding rule scanner, can you verify?